ktor-features/ktor-auth/jvm/src/io/ktor/auth/OAuth.kt

```diff
@@ -125,7 +126,8 @@ public sealed class OAuthServerSettings(public val name: String, public val vers
 
         public val authorizeUrlInterceptor: URLBuilder.() -> Unit = {},
         public val passParamsInURL: Boolean = false,
-        public val accessTokenInterceptor: HttpRequestBuilder.() -> Unit = {}
+        public val accessTokenInterceptor: HttpRequestBuilder.() -> Unit = {},
+        public val extraParameters: Map<String, String> = emptyMap()
```

Comment:
nit: I would add the `extraParameters` before `accessTokenInterceptor`. Not sure whether it would break existing users though.